test/Microsoft.Azure.Devices.Edge.Test.Common/IEdgeDaemon.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common
+{
+    using System;
+    using System.ServiceProcess;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public enum EdgeDaemonStatus
+    {
+        Running = ServiceControllerStatus.Running,
+        Stopped = ServiceControllerStatus.Stopped
+    }
+
+    public interface IEdgeDaemon
+    {
+        Task InstallAsync(string deviceConnectionString, Option<string> packagesPath, Option<Uri> proxy, CancellationToken token);
```

Comment:
nit: suggest to wrap these argument in 1 type; this will make this interface more stable and easier to add more parameter in the future.

Reply:
Yes this is an interface, but it's a test interface for internal use. We don't need to worry about versions or interface stability--certainly not at this point (with only one test). I think it would be premature to do something like that.